### PR TITLE
premium internals now increase the slots of your internals box

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
premium internals boxes from the station trait now get 2 more slots and have a slightly bigger sprite
also lowers the trait's weight a bit, making it equal to most other traits.
image

## Why It's Good For The Game
the main reason im doing this is because this trait messes with anything people wanna do to survival boxes, cause it takes up 2 slots, so you have to account for that whenever youre making a subtype that you wanna fill with more stuff, you cant really do that (currently an issue with the nukie medical pr)
the other reason is station traits should affect the gameplay a bit more than "wow i have 2 items i can normally get from an emergency toolbox", a larger box doesnt change that much but its something.

## Changelog
:cl:
balance: premium internals station trait now increases the slots of your internals box
/:cl:

### CODE REVIEW

**Comment by Xander3359** [comment]
Seems like an unnecessary buff. Not every station trait has to be super impactful and are typically discarded anyways. People just dump the wallets/scarves/flare+radio when they spawn in. This is purely just a buff but I dont even think it's warranted.

**Comment by Fikou** [comment]
> Seems like an unnecessary buff. Not every station trait has to be super impactful and are typically discarded anyways. People just dump the wallets/scarves/flare+radio when they spawn in. This is purely just a buff but I dont even think it's warranted.

this is the dumbest post ive read today

**Comment by Oranggu** [comment]
needless buff? maybe. do i want it to be added? yes.

**Comment by moocowswag** [comment]
> Seems like an unnecessary buff. Not every station trait has to be super impactful and are typically discarded anyways. People just dump the wallets/scarves/flare+radio when they spawn in. This is purely just a buff but I dont even think it's warranted.

Placebo buff, if you give people a random special bonus like inventory space itl trigger neuron activation and make them enjoy a round more.

**Comment by GuillaumePrata** [comment]
My first reaction was "Nooooooo! more inventory creep!!!1!"
Then I realize some people will beat each other and loot the bigger boxes which is just too funny of an interaction.

**Comment by Serxule** [comment]
wallets are actually helpful, scarves take up a (mostly) cosmetic item slot, but yeah no one uses the flare/radio it just takes up space

File: code/game/objects/items/storage/boxes/job_boxes.dm [hunk 1]
```diff
@@ -13,6 +13,14 @@
 	/// What medipen should be present in this box?
 	var/medipen_type = /obj/item/reagent_containers/hypospray/medipen
 
+/obj/item/storage/box/survival/Initialize(mapload)
```
[the remark this answers is not in the document]
Comment (Fikou) [reply code/game/objects/items/storage/boxes/job_boxes.dm:16]: you cant craft survival boxes

[the remark this answers is not in the document]
Comment (Fikou) [reply code/game/objects/items/storage/boxes/job_boxes.dm:16]: wtf u can thats fucking stupid

**Comment by Stonetear** [comment]
Perhaps you should make the box unflattenable.

**Comment by Fikou** [comment]
> Perhaps you should make the box unflattenable.

not as funny

**Comment by private-tristan** [comment]
traitorbiginternals

**Pull request merged**